Makefile_packaging.mk

```diff
@@ -324,20 +329,20 @@ chrootbuild: $(SRPM) $(CALLING_MAKEFILE)
 name=$$repo:$$branch:$$build_number\n\
 baseurl=$${JENKINS_URL:-https://build.hpdd.intel.com/}job/daos-stack/job/$$repo/job/$$branch/$$build_number/artifact/artifacts/centos7/\n\
 enabled=1\n\
-gpgcheck = False\n" >> /etc/mock/default.cfg;                                        \
+gpgcheck = False\n" >> /etc/mock/default.cfg;                                    \
 	    done;                                                                    \
-	    for repo in $(el7_LOCAL_REPOS) $(el7_REPOS); do                          \
+	    for repo in $($(DISTRO_BASE)_LOCAL_REPOS) $($(DISTRO_ID_BASE)_REPOS); do \
 	        repo_name=$${repo##*://};                                            \
 	        repo_name=$${repo_name//\//_};                                       \
 	        echo -e "[$$repo_name]\n\
 name=$${repo_name}\n\
 baseurl=$${repo}\n\
-enabled=1\n" >> /etc/mock/default.cfg;                                               \
+enabled=1\n" >> /etc/mock/default.cfg;                                           \
```

Comment:
Ditto.

Reply:
That looks intentional in the existing code to keep the trailing whitespace out of the /etc/mock/default.cfg.

Reply:
> That looks intentional in the existing code to keep the trailing whitespace out of the /etc/mock/default.cfg.

The lines above 340 have an intentional mis-alignment of the line continuator, but line 340 is just part of the general block above those lines.

I wouldn't make such a big deal about line continuators, and would just fix them myself on a subsequent patch, but this file is going to be used in `make packaging_check` diffs with every other packaging project, so we want to make sure we get it right.